cli/commands/rewards/generatemerkle.ts

```diff
@@ -92,11 +92,22 @@ export default class CalculateRewards extends BaseCommand {
     }
 
     const progressBar = cli.progress()
-    progressBar.start(allEvents.length, 0)
-    
-    for(let index in allEvents) {
+    progressBar.start(attestationEvents.length + transferEvents.length, 0)
+
+    attestationEvents.forEach(event => {
       progressBar.increment()
-      const event = allEvents[index]
+      if (event.event == 'AttestationCompleted') {
```

Comment:
Should we define these as an enum somewhere?

Reply:
this would be cleaner, yes. Will do this.